standalone/package.json

```diff
@@ -0,0 +1,90 @@
+{
+	"name": "@guardian/commercial-standalone",
+	"version": "0.0.0-this-never-updates-in-source-code-refer-to-git-tags",
+	"description": "Guardian advertising business logic",
+	"homepage": "https://github.com/guardian/commercial-core#readme",
+	"bugs": {
+		"url": "https://github.com/guardian/commercial-core/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/guardian/commercial-core.git"
+	},
+	"license": "Apache-2.0",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"files": [
+		"dist"
+	],
+	"scripts": {
+		"build": "webpack -c webpack.config.prod.js",
+		"build:dev": "webpack -c webpack.config.dev.js",
+		"serve": "OVERRIDE_BUNDLE=true yarn webpack-dev-server -c ./webpack.config.dev.js"
+	},
+	"prettier": "@guardian/prettier",
+	"devDependencies": {
+		"@commitlint/cli": "^17.0.3",
+		"@commitlint/config-conventional": "^17.0.0",
+		"@guardian/ab-core": "^2.0.0",
+		"@guardian/consent-management-platform": "^10.6.0",
+		"@guardian/eslint-config-typescript": "^1.0.0",
+		"@guardian/libs": "^7.1.3",
+		"@guardian/prettier": "^2.1.1",
+		"@octokit/core": "^4.0.5",
+		"@semantic-release/github": "^8.0.2",
+		"@types/google.analytics": "^0.0.42",
+		"@types/googletag": "^2.0.0",
+		"@types/jest": "^27.0.3",
+		"@typescript-eslint/eslint-plugin": "^5.5.0",
+		"@typescript-eslint/parser": "^5.5.0",
+		"babel-loader": "^9.1.0",
+		"circular-dependency-plugin": "^5.2.2",
+		"commitizen": "^4.2.4",
+		"conventional-changelog-conventionalcommits": "^5.0.0",
+		"cz-conventional-changelog": "^3.3.0",
+		"eslint": "^8.4.1",
+		"eslint-config-prettier": "^8.3.0",
+		"eslint-plugin-eslint-comments": "^3.2.0",
+		"eslint-plugin-import": "^2.25.3",
+		"eslint-plugin-jest": "^26.1.5",
+		"eslint-plugin-prettier": "^4.0.0",
+		"husky": "^8.0.1",
+		"jest": "^27.4.1",
+		"lint-staged": "^13.0.3",
+		"mockdate": "^3.0.5",
+		"npm-run-all": "^4.1.5",
+		"prettier": "^2.5.0",
+		"semantic-release": "^19.0.2",
+		"ts-jest": "^27.0.7",
+		"ts-loader": "^8.2.0",
+		"type-fest": "^2.8.0",
+		"typescript": "^4.5.2",
+		"uglifyjs-webpack-plugin": "^2.2.0",
+		"web-vitals": "^2.1.2",
+		"webpack": "^5.75.0",
+		"webpack-bundle-analyzer": "^3.3.2",
+		"webpack-cli": "^4.10.0",
+		"webpack-dev-server": "^4.9.3",
+		"webpack-merge": "^4.1.0"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"peerDependencies": {
+		"@guardian/ab-core": "^2.0.0",
+		"@guardian/libs": ">6"
+	},
+	"dependencies": {
+		"@guardian/commercial-core": "^4.25.0",
+		"@guardian/source-foundations": "^7.0.1",
+		"fastdom": "^1.0.11",
+		"lodash-es": "^4.17.21",
+		"ophan-tracker-js": "^1.4.0",
+		"prebid.js": "guardian/prebid.js#2e3b96d",
+		"process": "^0.11.10",
+		"raven-js": "^3.27.2",
+		"raw-loader": "^4.0.2",
+		"svg-loader": "^0.0.2",
+		"wolfy87-eventemitter": "^5.2.9"
```

Comment:
Would this result in DCR / Frontend shipping 2 instances of common dependencies such as source-foundations, fastdom, ophan, raven, etc – once in the commercial bundle and once in the standard JS for these platforms? Perhaps this is already happening, I'm not sure